src/chapters/git/remote-repository.rst

```diff
@@ -0,0 +1,182 @@
+Remote Repositories
+===================
+
+In this chapter, we learned how to use the Git version control system to keep
+backups of our code. This version history (the repository) is stored on our
+personal machine.
+
+One advantage of keeping the local repo is that we can work on our project
```

Comment:
Have we defined "local"?

Reply:
If not, this would be a natural place to do so..

Reply:
"Local development" was defined in chapter 16 when we introduced the Terminal.